### DISCUSSION TITLE
使用自定义应用获取open token 时报错

### DISCUSSION THREAD

**Discussion by muxin-403** [AlistGo/alist#6138] [Q&A]
使用自定义应用使用Get Aliyundrive Refresh Token工具获取 open token 时报错
![image](https://github.com/alist-org/alist/assets/125569763/ed90a0f7-5d54-434d-83fc-274b0fd7fced)
大家有遇到这种情况吗？

**Comment by 520yxl** [comment]
> 使用自定义应用使用Get Aliyundrive Refresh Token工具获取 open token 时报错
> ![image](https://github.com/alist-org/alist/assets/125569763/ed90a0f7-5d54-434d-83fc-274b0fd7fced)
> 大家有遇到这种情况吗？

没有设置回调地址

现已支持使用自己的client，在此处获取：https://alist.nn.ci/tool/aliyundrive/request.html
回调地址为： https://alist.nn.ci/tool/aliyundrive/callback
添加存储时，将client_id和client_secret也填上即可。
